src/Symfony/Component/Messenger/Bridge/Doctrine/Transport/PostgreSqlConnection.php

```diff
@@ -26,7 +26,7 @@ final class PostgreSqlConnection extends Connection
 {
     /**
      * * check_delayed_interval: The interval to check for delayed messages, in milliseconds. Set to 0 to disable checks. Default: 60000 (1 minute)
-     * * get_notify_timeout: The length of time to wait for a response when calling PDO::pgsqlGetNotify, in milliseconds. Default: 0.
+     * * get_notify_timeout: The length of time to wait for a response when calling PDO::pgsqlGetNotify (or Pdo\Pgsql::getNotify on PHP 8.5+), in milliseconds. Default: 0.
```

Comment:
The new subclasses have been introduced in PHP 8.4.

```suggestion
     * * get_notify_timeout: The length of time to wait for a response when calling PDO::pgsqlGetNotify (or Pdo\Pgsql::getNotify on PHP 8.4+), in milliseconds. Default: 0.
```